guides/12-general/8-get-your-token-verified.md

```diff
@@ -0,0 +1,31 @@
+---
+sidebar_label: "Get Your Token Verified"
+description: Learn more about how to get your token the verified badge on Jupiter.
+slug: "/get-your-token-verified"
+title: Verify Your Token on Jupiter
+---
+
+<head>
+    <title>How to Get Your Token Verified on Jupiter</title>
+    <meta name="twitter:card" content="summary" />
+</head>
+
+![verify-token-banner](../img/verify-token-banner.png)
+
+All unverified tokens are marked with a **'⚠️’** label to encourage users to double-check that the mint addresses are the right ones before trading.
+
+If you've created a token and want to get it verified on the Jupiter Ecosystem List, you can submit your token for review on [catdetlist.jup.ag](http://catdetlist.jup.ag).
+
+For approval, your project needs to:
+
+- **Be at least 21 days old**
+- **Have daily volume of more than 10K**
+- **Have a market cap of more than 100K**
+- **Have unique metadata (Symbol and Name does not duplicate another project on the community list)**
+- **Have more than 500 unique holders**
+
+:::tip Note
+Your token does not have to be verified to be tradable. All new pools are supported for instant routing for **14 days** before pools get re-evaluated. Remind your users to search via your mint address so they can swap safely!
+:::
+
+More details about the community tag initiative in [this post](https://www.jupresear.ch/t/get-your-token-a-community-tag/18963).
```

Comment:
Need more information from this [channel](https://discord.com/channels/897540204506775583/1108064584654405642)

Contains FAQ, etc too